src/lib/y2users/help_texts.rb

```diff
@@ -47,7 +46,15 @@ def validation_config
     #
     # @return [String] formatted and localized text
     def valid_password_text
-      Yast::UsersSimple.ValidPasswordHelptext
+      # NOTE: to avoid dependencies, this help text is a duplicate of
+      #   Yast::UsersSimple.ValidPPasswordHelpText. Please, keep them in sync.
```

Comment:
ValidPPassword, double "P"

Reply:
:see_no_evil: I'll hide your comment right away! It never happened :stuck_out_tongue_winking_eye: